pkg/preview/preview.go

```diff
@@ -0,0 +1,71 @@
+package preview
+
+import (
+	"log"
+	"os"
+	"strconv"
+
+	"github.com/fatih/color"
+	"github.com/joho/godotenv"
+	"github.com/spf13/cobra"
+)
+
+type devPreview struct {
+	commands  []*cobra.Command
```

Comment:
what are commands for

Reply:
`commands` stores all the commands that have been added for dev preview by the host CLI developer.

I've added comments for the same in the file now, please have a look.